src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -173,88 +198,116 @@ subroutine readParams ( ncid )
     type(file_desc_t),intent(inout) :: ncid   ! pio netCDF file id
     !
     ! !LOCAL VARIABLES:
-    character(len=32)  :: subname = 'CNDecompBgcParamsType'
-    character(len=100) :: errCode = 'Error reading in CN const file '
+    character(len=32)  :: subname = 'readParams'
+    character(len=100) :: errCode = 'Error reading MIMICS params '
     logical            :: readv   ! has variable been read in or not
     real(r8)           :: tempr   ! temporary to read in constant
     character(len=100) :: tString ! temp. var for reading
     !-----------------------------------------------------------------------
 
     ! Read off of netcdf file
-    tString='tau_l1'
+    ! TODO slevis: new params will be added here and in the params file
+    ! TODO slevis: Keep tau_cwd?
+    tString='tau_cwd'
```

Comment:
This and the next few are not new. The differencing code interpreted them as new.

Reply:
see comment above on C:N ratios